src/api/app/views/webui2/webui/package/_breadcrumb_items.html.haml

```diff
@@ -0,0 +1,6 @@
+= render partial: 'webui/project/breadcrumb_items'
+%li.breadcrumb-item
+  = link_to @package, package_show_path(@package)
```

Comment:
This is missing the project and `link_to @object` is a link already so this becomes `/package/show/$package/$package`

Reply:
Done.